### PR TITLE
live button feature

### PR DESCRIPTION
**Pull request by vinnyjth**

### CODE REVIEW

File: apollos-church-api/src/data/craft/ContentItem.js [hunk 1]
```diff
@@ -1172,6 +1191,7 @@ export class dataSource extends CraftDataSource {
     // if there is no live stream, then there is no live content. Easy enough!
     // if (!isLive) return [];
     const mostRecentSermon = await this.getMostRecentSermon();
+    console.log(mostRecentSermon, 'mostRecentSermon');
```
Comment (vinnyjth) [inline apollos-church-api/src/data/craft/ContentItem.js:1194]:
```suggestion

```

**Comment by conrad-vanl** [comment]
preview-full-2021-09-16 12 19 05
 
@adamhann  this is step 1 of the requests that you've asked for - this will add the "Join us for church online" button on the live content items. Additionally (and @vinnyjth correct me if I'm wrong - I know we talked about a couple of different options here) this button will display all the time for the current "live item". In other words, even if you open this screen up outside of a gathering time, this button will still show up on the sermon that will be going live. We thought that would be better as that gives people the ability to open CHOP up, see the schedule, and know where and when to go when they come back.

Are you good with us pushing this out live or do you want us to wait?

**Comment by adamhann** [comment]
Hey Guys, a couple of feedback items on change. 

- Is it possible to make the button a little more "live" looking. I think in all of our heads we thought the button would be red or demand more attention so people know that it's a button. This ties to my second point. 
- We'd only like for this to show up within a certain time window. On our website we are making the live button show up 10 minutes before the gathering starts and for up to 15 minutes after the gathering starts. We wouldn't want this button on all the time. Is that possible?
- Could we also change the language? Maybe something like "Join us live for Church Online".

**Comment by vinnyjth** [comment]
@adamhann  We have updated the language on the button, and caused the button to only show up when the CHoP livestream is live. Can we  go ahead and release this? 

Changing the button style so it shows up more distinctly will have to involve a further front end update.

**Comment by adamhann** [comment]
@vinnyjth Yes, we can go ahead and push. I know we'd like to make the button more distinct but we can wait on that until you guys have time to make the front-end update. Thanks for working on this!

**Pull request merged**